<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>570</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>DeepOrder</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #EAEFEF;</string>
  </property>
  <widget class="QWidget" name="mainwindow">
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QFrame" name="frame_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>550</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
border:1px solid black;
border-radius: 5px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>478</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>128</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: darkgray;
color: white;
border: none;
border-top-left-radius :5px;
border-top-right-radius : 5px;
border-bottom-left-radius : 0px;
border-bottom-right-radius : 0px;</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>DeepOrder</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>420</width>
       <height>300</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
border-radius: 5px;
QListWidget::item {
        height: 40px;
    }</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDropMode::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::DropAction::CopyAction</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Movement::Static</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::LayoutMode::SinglePass</enum>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="viewMode">
      <enum>QListView::ViewMode::ListMode</enum>
     </property>
     <property name="itemAlignment">
      <set>Qt::AlignmentFlag::AlignBaseline</set>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>macro1</string>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
        <stylestrategy>NoAntialias</stylestrategy>
       </font>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
border-radius: 5px;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="placeholderText">
      <string>이름을 입력하세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_add">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>400</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color :#f0f0f0;;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_delete">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #f0f0f0;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_edit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>440</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #f0f0f0;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>수정</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_run">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>150</width>
       <height>70</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>128</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: darkgray;
color: white;
border-top: 1px solid black;
border-bottom: 1px solid black;
border-left: 1px solid black;
border-right: none;
border-top-left-radius :5px;
border-top-right-radius : 0px;
border-bottom-left-radius : 5px;
border-bottom-right-radius : 0px;</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>RUN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_stop">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>440</y>
       <width>150</width>
       <height>70</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>128</horstretch>
       <verstretch>60</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: deepskyblue;
color: white;
border-top: 1px solid black;
border-bottom: 1px solid black;
border-left: none;
border-right: 1px solid black;
border-top-left-radius :0px;
border-top-right-radius : 5px;
border-bottom-left-radius : 0px;
border-bottom-right-radius : 5px;</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_version">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>130</width>
       <height>10</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color:black;</string>
     </property>
     <property name="text">
      <string>v1.0.0 updated 250201</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_copy">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>440</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #f0f0f0;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>복제</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_setting">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>480</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferVerticalHinting</hintingpreference>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #f0f0f0;
    padding: 2px 4px;
}</string>
     </property>
     <property name="text">
      <string>설정</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::InsertText"/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
